src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/hotheap/HighHeapUsageOldGenRca.java

```diff
@@ -80,11 +92,12 @@
   // minimum
   private static final double OLD_GEN_GC_THRESHOLD = 1;
   private static final double CONVERT_BYTES_TO_MEGABYTES = Math.pow(1024, 3);
+  private static final int TOP_K = 3;
   protected Clock clock;
 
 
   public <M extends Metric> HighHeapUsageOldGenRca(final int rcaPeriod, final double lowerBoundThreshold,
-      final M heap_Used, final M gc_event, final M heap_Max) {
+      final M heap_Used, final M gc_event, final M heap_Max, final List<Metric> consumers) {
     super(5);
```

Comment:
We should make that change where this is no longer required to be specified and it becomes a constant in the `Node` class

Reply:
I agree. But that will be a big code change which involves a lot of files and unit tests. Let me create a github issue to keep track. we might need to create a separate PR to address it.